.gitignore

```diff
@@ -25,3 +25,7 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+
+# Serverless directories
+.serverless
```

Comment:
no need to add it a second time, as it is already there by default (see line#13)